### PR TITLE
Recreate invalidated ImageViews caused by Image destruction

### PR DESCRIPTION
**Pull request by purvisa-at-google-com**

### CODE REVIEW

File: gapis/api/vulkan/frame_loop.go [hunk 1]
```diff
@@ -815,7 +815,7 @@ func (f *frameLoop) detectChangedBuffers(ctx context.Context) {
 			poolID := data.Pool()
 
 			// Did we see this buffer get written to during the loop? If we did, then we need to capture the values at the start of the loop.
-			// TODO: This code does not handle the possibility of new DeviceMemory being bound to the object during the loop. Talk to @purvisa.
+			// TODO: This code does not handle the possibility of new DeviceMemory being bound to the object during the loop. TODO(purvisa).
```
Comment (hevrard) [inline gapis/api/vulkan/frame_loop.go]: I meant adding your name in the TODO at the beginning of the comment :)

File: gapis/api/vulkan/frame_loop.go [hunk 2]
```diff
@@ -869,7 +869,7 @@ func (f *frameLoop) detectChangedImages(ctx context.Context) {
 						poolID := data.Pool()
 
 						// Did we see this part of this image get written to during the loop? If we did, then we need to capture the values at the start of the loop.
-						// TODO: This code does not handle the possibility of new DeviceMemory being bound to the object during the loop. Talk to @purvisa.
+						// TODO: This code does not handle the possibility of new DeviceMemory being bound to the object during the loop. TODO(purvisa).
```
Comment (hevrard) [inline gapis/api/vulkan/frame_loop.go]: idem

File: gapis/api/vulkan/frame_loop.go (hunk 1 above)
Comment (pau-baiget) [reply gapis/api/vulkan/frame_loop.go]: I'm not a fan of adding people's usernames (specially if they don't match the Github one). Whoever has questions around this code should just "git blame" and ask the corresponding person.

Comment (hevrard) [reply gapis/api/vulkan/frame_loop.go]: This `TODO(username)` is Google style recommendation:  https://google.github.io/styleguide/cppguide.html#TODO_Comments

And it is already quite used in this codebase, try `git grep 'TODO('` to see many instances.

So I think it is worth sticking to this style for consistency.

Comment (pau-baiget) [reply gapis/api/vulkan/frame_loop.go]: Woopsie! sorry :S

**Pull request merged**